__tests__/app.test.js

```diff
@@ -0,0 +1,57 @@
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const { topicData, userData, articleData, commentData  } = require("../db/data/test-data");
+
+beforeEach(() => {
+    return seed({ topicData, userData, articleData, commentData });
+});
+  
+afterAll(() => {
+    return db.end();
+});
+
+describe("app", () => {
+ describe("GET /api/topics", () => {
+  test("200:responds with a status of 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("200: responds with an array of topics with correct length", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then((response) => {
+      const {topics} = response.body;
+      const expectedLength = topicData.length;
+      expect(topics).toHaveLength(expectedLength);
+    });  
+  });
+  test("200: responds with an array of topic objects that each have the properties slug, description", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then((response) => {
+      const {topics} = response.body;
+      topics.forEach((topics) => {
```

Comment:
Just a convention thing, but usually if we have a ```.map``` or ```.forEach``` or anything that looks at one item of many items, we use the singular form (i.e. 'topic') in the smooth brackets, because it represents the one topic that we are looking at each time. It's good practice to implement 🙂